Task1/ClassLibrary/Logger.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace LoggerClassLibrary
+{
+    public enum LogLevel
+    {
+        Info,
+        Warning,
+        Error
+    }
+    public class Logger: ILogger
+    {
+        Dictionary<string, ILogger> destinationNameToObjectDictionary = new Dictionary<string, ILogger>();
+        Dictionary<LogLevel, List<string>> logLevelToDestinationsDictionary;
```

Comment:
Can we have one dictionary `Dictionary<LogLevel, List<ILogger>>` instead of two?